kyuubi-server/web-ui/public/enable.html

```diff
@@ -0,0 +1,29 @@
+<!--
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Apache Kyuubi</title>
+  </head>
+  <body>
+    <h2 style="text-align: center;">🙁 The Web UI is currently unavailable.</h2>
+    <h4 style="text-align: center;">👉 To enable Web UI, set kyuubi.frontend.rest.enable.webui true.</h4>
```

Comment:
the page may not much help as we did for `dist` missing `--web-ui`, let's keep 404 now